scripts/python/update_params.py

```diff
@@ -0,0 +1,65 @@
+"""Update parameters files to latest."""
```

Comment:
What versions of BciPy would this support?

Reply:
I think this works for all versions.